src/mbgl/layout/symbol_layout.cpp

```diff
@@ -90,10 +92,12 @@ SymbolLayout::SymbolLayout(std::string bucketName_,
                 u8string = platform::lowercase(u8string);
             }
 
-            ft.text = util::utf8_to_utf32::convert(u8string);
+            std::u16string u16string = util::utf8_to_utf16::convert( u8string );
```

Comment:
@jfirebaugh uncomment [this line](https://github.com/mapbox/mapbox-gl-native/blob/master/scripts/clang-tools.sh#L52) and run `make check`. This will run both `clang-tidy` and `clang-format` on modified files.